modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaCommerceAccountGroupRelLocalServiceCheck.java

```diff
@@ -0,0 +1,112 @@
+package com.liferay.source.formatter.check;
+
+import com.liferay.petra.string.StringBundler;
+import com.liferay.petra.string.StringPool;
+import com.liferay.petra.string.StringUtil;
+import com.liferay.source.formatter.check.util.JavaSourceUtil;
+import com.liferay.source.formatter.parser.JavaClass;
+import com.liferay.source.formatter.parser.JavaClassParser;
+import com.liferay.source.formatter.parser.JavaMethod;
+import com.liferay.source.formatter.parser.JavaTerm;
+
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class UpgradeJavaCommerceAccountGroupRelLocalServiceCheck
+	extends BaseUpgradeCheck {
+
+	@Override
+	protected String format(
+		String fileName, String absolutePath, String content)
+		throws Exception {
+
+		JavaClass javaClass = JavaClassParser.parseJavaClass(fileName, content);
+
+		for (JavaTerm childJavaTerm : javaClass.getChildJavaTerms()) {
+			if (!childJavaTerm.isJavaMethod()) {
+				continue;
+			}
+
+			JavaMethod javaMethod = (JavaMethod)childJavaTerm;
+
+			String javaMethodContent = javaMethod.getContent();
+
+			String newJavaMethodContent = _replaceCommerceAccountGroupRel(
+				javaMethodContent, content);
+
+			content = StringUtil.replace(
+				content, javaMethodContent, newJavaMethodContent);
+		}
+
+		return content;
```

Comment:
@lazhcsilva, check if the SF allows you to do the following:

```
		JavaClass javaClass = JavaClassParser.parseJavaClass(fileName, content);

		String newContent = content;

		for (JavaTerm childJavaTerm : javaClass.getChildJavaTerms()) {
			if (!childJavaTerm.isJavaMethod()) {
				continue;
			}

			JavaMethod javaMethod = (JavaMethod)childJavaTerm;

			String javaMethodContent = javaMethod.getContent();

			newContent = StringUtil.replace(
				newContent, javaMethodContent, 
				_replaceCommerceAccountGroupRel(
					javaMethodContent, newContent));
		}

		return content;
```
You are only using the newJavaMethodContent on the replace, if wasn't the SF that requested the variable we should use it directly.